src/JuliaFormatter.jl

```diff
@@ -844,7 +844,13 @@ function format(paths; options...)::Bool
             else
                 options
             end
-            _format_file(path; opts...)
+            try # log which file fails
+                # https://github.com/domluna/JuliaFormatter.jl/issues/640
+                _format_file(path; opts...)
+            catch
+                @info "Error in formatting file $path"
+                _format_file(path; opts...)
+            end
```

Comment:
Thoughts on (optionally) showing the original exception? E.g.
```suggestion
            catch
                @info "Error in formatting file $path"
                @debug "formatting failed due to" exception=(err, catch_backtrace()
                _format_file(path; opts...)
            end
```